docs/Introduction/tutorials/advanced-tutorial.md

```diff
@@ -11,47 +11,64 @@ metadata:
-- Be able to find the Oracle Jobs and Adapters for your contract
-- How to request data from an Oracle Job
+In this tutorial you will learn how to request data from a public API in a smart contract. This includes understanding what Core adapters and External adapters are and how Oracle Jobs use them. You will also learn how to find the Oracle Jobs and Adapters for your contract and how to request data from an Oracle Job.
 
-# 1. Requesting API Data
+**Table of Contents**
 
-## 1a. Request & Receive Recap
++ [1. How does the request and receive cycle work for API calls?](#1-how-does-the-request-and-receive-cycle-work-for-api-calls)
++ [2. What are initiators?](#2-what-are-initiators)
++ [3. What are Adapters?](#3-what-are-adapters)
+  + [Core Adapters](#core-adapters)
+  + [Contract Example](#contract-example)
+  + [External Adapters](#external-adapters)
++ [4. How can I use Adapters in my own contract?](#4-how-can-i-use-adapters-in-my-own-contract)
+  + [Variables](#variables)
+  + [Constructor](#constructor)
+  + [`requestData` Function](#requestdata-function)
+  + [Callback Function](#callback-function)
++ [5. How do I deploy to testnet?](#5-how-do-i-deploy-to-testnet)
++ [6. Further Reading](#6-further-reading)
+
+# 1. How does the request and receive cycle work for API calls?
 
 The request and receive cycle describes how a smart contract requests data from an oracle and receives the response in a separate transaction. If you need a refresher, check out the [Basic Request Model](../architecture-request-model/).
 
-In the [Random Numbers tutorial](../intermediates-tutorial/), we request randomness from a VRF oracle, then await the response. The fulfilment function is already given to us from the `VRFConsumerBase` contract, so oracles already know where to send the response to. However, with API calls, our contract _defines_ which function it wants to receive the response to.
+In the [Random Numbers](../intermediates-tutorial/) tutorial, you requested randomness from a VRF oracle, then await the response. The fulfilment function is already given to us from the `VRFConsumerBase` contract, so oracles already know where to send the response to. However, with API calls, the contract itself *defines* which function it wants to receive the response to.
+
+Before creating any code, you should understand how Oracle jobs can get data on-chain.
 
-However, before we go into the implementation, let's first understand how Oracle jobs can get data on-chain.
+# 2. What are initiators?
 
-## 1b. Initiators
+[**Initiators**](../initiators/) are what start, or initiate, a job inside an Oracle. In the case of a Request and Receive job, the [RunLog](/docs/initiators/#runlog) initiator monitors the blockchain for a request from a smart contract. Once it catches a request, it initiates the job. This runs the adapters (both core and external) that the job is configured to run eventually returning the response to the requesting contract.
```

Comment:
Internal Initiators have been "sort of" replaced by "type" of jobs. And "runlog" type/initiator has is now direct request jobs. https://docs.chain.link/docs/jobs/types/direct-request/ as runlog has been deprecated.

Could we add this context here?

Reply:
Captured here: https://github.com/smartcontractkit/documentation/issues/500

We'll address this in a followup.